content/09.overfitting.md

```diff
@@ -16,9 +16,11 @@ Additionally, there are a variety of techniques to reduce overfitting during tra
 Another way, as described by Chuang and Keiser, is to identify the baseline level of memorization of the network by training on the data with the labels randomly shuffled and to see if the model performs better on the actual data [@doi:10.1021/acschembio.8b00881].
 If the model performs no better on real data than randomly scrambled data, then the performance of the model can be attributed to overfitting.
 
-Additionally, one must be sure that their dataset is not skewed or biased, such as by having confounding and scientifically irrelevant variables that the model can pick up on [@doi:10.1371/journal.pmed.1002683].
-In this case, simply holding out test data is insufficient.
+Additionally, one must be sure that their dataset is not skewed or biased, such as by having confounding and scientifically irrelevant variables that the model can pick up on.
```

Comment:
Do we want to say _independent_ and identically distributed? It may be helpful to include the slightly more technical terminology here as it would connect to the broader literature on the topic.

Reply:
For ensuring generalization, is identical distribution a hard requirement? In the hospital example, independent hospitals had different rates of pneumonia which should not have affected the model if it were truly learning the signature of pneumonia rather than the relative frequencies.

Reply:
I definitely think independent is the bigger issue here. Identically distributed is helpful for performance but if your test set is not identically distributed in the same way that other generalization sets are not, I guess you should still get an accurate performance estimate.

Reply:
Ok this is fixed as:

> Additionally, in biology and medicine it is critical to consider independence when defining training and test sets.